src/public/modules/forms/helpers/decryption.worker.js

```diff
@@ -150,9 +152,8 @@ async function decryptIntoCsv(data) {
           }
         })
 
-        downloadBlob = await downloadAndDecryptAttachmentsAsZip(
-          attachmentDownloadUrls,
-          secretKey,
+        downloadBlob = await queue.add(() =>
+          downloadAndDecryptAttachmentsAsZip(attachmentDownloadUrls, secretKey),
```

Comment:
to add try-catch block to isolate attachment failure from CSV decryption failure

Reply:
Added